src/ui/CodeRenderer/DiffLine/DiffLine.tsx

```diff
@@ -1,18 +1,31 @@
 import cs from 'classnames'
-import PropTypes from 'prop-types'
 
 import {
   classNamePerLineContent,
   classNamePerLineState,
   getLineState,
-  LINE_TYPE,
   lineStateToLabel,
+  type LineType,
 } from 'shared/utils/fileviewer'
 import CoverageLineIndicator from 'ui/CodeRenderer/CoverageLineIndicator'
 
 import { useScrollToLine } from '../hooks'
 
-function DiffLine({
+type LineContent = { types: Array<string>; content: string }
+
+export interface DiffLineProps {
+  lineContent?: Array<LineContent>
```

Comment:
curious on this changing from a required prop when we were using PropTypes to optional

Reply:
Typo